projects/archiver/src/tasks/indexer/helpers/summary.spec.ts

```diff
@@ -19,6 +19,28 @@ describe('getOtherRecentIssues', () => {
             { edition: usEdition, issueDate: '2019-10-06' },
```

Comment:
Small improvement but you could probably write a small function to generate this.

Reply:
I think the more you abstract away in tests, the more you need to then test that abstraction. 

I do hate it... but i think it needs to stay or be changed.